### PR TITLE
(examples): add nextjs-vercel-aws-cdk-example

### PR DESCRIPTION
**Pull request by vvo**
This adds an example PostgreSQL (RDS), EventBridge (crons) and SNS (background jobs) stack along with a Next.js application

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ This section includes code libraries in various programming languages which vend
 * [Example of REST API built with CDK](https://github.com/shaftoe/api-l3x-in) - Source code that powers REST APIs at https://api.l3x.in/.
 * [dilbert-feed](https://github.com/mlafeldt/dilbert-feed) - A serverless application written in Go that allows you to enjoy Dilbert in your RSS feed reader without any ads.
 * [django-postgres-vue-gitlab-ecs](https://gitlab.com/verbose-equals-true/django-postgres-vue-gitlab-ecs) - An example Django + Vue.js web app deployed with CDK using GitLab CI.
+* [nextjs-vercel-aws-cdk-example](https://github.com/vvo/nextjs-vercel-aws-cdk-example) - A PostgreSQL (RDS), EventBridge (crons) and SNS (background jobs) example along with a Next.js application
```
Comment (vvo) [inline readme.md:127]:
```suggestion
* [nextjs-vercel-aws-cdk-example](https://github.com/vvo/nextjs-vercel-aws-cdk-example) - A PostgreSQL (RDS), EventBridge (crons) and SNS (background jobs) example along with a Next.js application.
```

**Comment by vvo** [comment]
Seems like build was broken previously.

**Comment by kolomied** [comment]
@vvo First, thanks for contribution! Build was fine previously - what happened is that new version of the linter was released with additional checks enabled (detected a couple of duplicate URLs, which is quite cool). I install the latest version during the build to keep the list up to date.

**Review by kolomied — APPROVED** [review]

**Comment by kolomied** [comment]
Please add "Mentioned in awesome" badge to your repo with the link to this list - [example](https://github.com/elegantdevelopment/aws-cdk-dynamodb-seeder).

Thanks for the contribution!

**Comment by vvo** [comment]
(done)

**Pull request merged**